src/IonParserBinaryRaw.ts

```diff
@@ -209,15 +209,22 @@ export class ParserBinaryRaw {
         return LongInt.fromVarIntBytes(bytes, isNegative);
     }
 
-    private readLongInt() : LongInt {
-        if (this._len - this._in.position() < 1) return new LongInt(0);
-        let bytes : Uint8Array = this._in.view(this._in.position(), );
-        let sign = (bytes[0] & 0x10) != 0;
-        bytes[0] &= 0x7F;
-        return LongInt.fromIntBytes(Array.prototype.slice.call(bytes), sign);
+    static readSignedIntFrom(input: BinarySpan, numberOfBytes: number) : LongInt {
```

Comment:
I removed the `private` keyword because I assumed that would prevent that method from being used in the unit tests, but apparently it doesn't. I'll restore it.

Reply:
yea nothings really private =(